cabal.project

```diff
@@ -29,6 +29,12 @@ packages:
          ./plugins/hls-selection-range-plugin
          ./plugins/hls-change-type-signature-plugin
          ./plugins/hls-gadt-plugin
+         ./plugins/hls-hident-plugin
+
+source-repository-package
```

Comment:
What do you need to remove this? Do you think it's going to happen soon? If not, add a comment explaining why this is here and when we can remove it.

Reply:
Done.